.circleci/config.yml

```diff
@@ -326,7 +326,7 @@ jobs:
           channel: '#sh_releases'
           color: '#2EB67D'
           title: $(cat .version)
-          message: https://github.com/codacy/chart/releases/tag/$(cat .version)/changelog.txt
+          message: https://github.com/codacy/chart/releases/tag/$(cat .version)
```

Comment:
This will simply post the link to the Slack channel, right?

I think we could give a bit more context on this message, for example:

```
Version $(cat .version) of the Codacy chart has been released!
Release on GitHub: https://github.com/codacy/chart/releases/tag/$(cat .version)
Release notes: https://github.com/codacy/chart/releases/download/$(cat .version)/$<release notes file name>
```

This would also link both to the release permalink on GitHub as well as to the release notes.

What do you think?